schema/config_blobset_blob.json

```diff
@@ -8,29 +8,20 @@
     "phase": {
       "$ref": "file:common.json#/definitions/phase"
     },
-    "content_type": {
-      "type": "string"
-    },
-    "base64": {
-      "type": "string",
-      "pattern": "^[0-9A-Za-z=+/]{4,}$"
-    },
     "url": {
-      "type": "string",
-      "format": "uri"
+      "type": "string"
     },
     "sha256": {
       "type": "string",
       "description": "Expected hash of the retrieved resource",
       "pattern": "^[0-9a-f]{64}$"
     },
-    "nonce": {
+    "generation": {
```

Comment:
How about "timestamp" which is less ambiguous and consistent to other format=date-time fields?

Reply:
The semantic meaning of "generation" is different than "timestamp" -- e.g. the value of the status message should match the generation of the config message -- it's NOT the timestamp of the status message/entry itself. The purpose of this is to tie the fields together, not indicating the time of the thing itself.  It's the same usage as the discovery subblocks use which have a similar semantic meaning.  We can sort through if that's the right behavior and/or term, but that's the intent behind it (and why the term is different).

Reply:
I created a doc so we can discuss this somewhere more nimble than here